_posts/2018-03-07-bug-bbq.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+permalink: 2018-03-07-bug-bbq
+subheadline: "Bug BBQ"
+title: "Announcing the Spring 2018 Bug BBQ"
+teaser: "Mark your calendar, the Spring 2018 Bug BBQ will take place April 12-13!"
+header:
+    image_fullwidth: "light-blue-wood-texture.jpg"
+categories:
+    - blog
+comments: true
+show_meta: true
+authors: ["François Michonneau", "Erin Becker"]
+---
+
+
+The Carpentries are preparing to publish the **Social Sciences** and the **Geospatial** curricula on [Zenodo](https://zenodo.org/) by mid-May. This will be the first 
```

Comment:
Social sciences lessons are scheduled for publication in April, Geospatial in June.